src/TextInput/TextInput.stories.tsx

```diff
@@ -11,29 +10,70 @@ export default {
 } as ComponentMeta<typeof TextInput>
 
 export const TextInputComponent: ComponentStory<typeof TextInput> = () => {
+  const [text, setText] = useState<string>('')
+
   return (
-    <ScrollView>
-      <TextInput label={'Password'} isRequire autoFocus secureTextEntry />
+    <ScrollView contentContainerStyle={{paddingBottom: 100}}>
+      <View style={{marginTop: 50}} />
+      <TextInput.Flat
+        label={'Icon left'}
+        value={text}
+        onChangeText={setText}
+        autoCapitalize={'words'}
+        autoCorrect
+        leftComponent={<TextInput.Icon source={require('../assets/folder-search.png')} />}
+        containerStyle={{marginHorizontal: 20}}
+        errorText={'sadasdsadsa'}
```

Comment:
Use a more descriptive content for these

Reply:
done